dotCMS/src/main/java/com/dotmarketing/db/LocalTransaction.java

```diff
@@ -147,7 +148,16 @@ static public void wrap(final VoidDelegate delegate) throws Exception {
             }
         }
     } // wrap.
-
+    
+    static public void wrapNoException(final VoidDelegate delegate)  {
+        try {
+            wrap(delegate);
+        }
+        catch(Exception e) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this "catch" on the same line that the previous closing curly brace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck)